snapcraft/tests/test_plugin_python.py

```diff
@@ -0,0 +1,249 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015 Canonical Ltd
```

Comment:
I personally prefer 2015-2016 as then it is just a one digit change to maintain, but I don't mind either.